src/engine/strat_engine/filesystem.rs

```diff
@@ -153,6 +153,21 @@ pub fn create_fs(dev_path: &Path) -> EngineResult<()> {
     Ok(())
 }
 
+pub fn grow_fs(dev_path: &Path) -> EngineResult<()> {
```

Comment:
It they are non-junk methods, they should be held to the same standard as our other "permanent" code.